.github/workflows/main.yml

```diff
@@ -71,6 +71,23 @@ jobs:
       run: >-
         conda build --output-folder=conda/pkg  conda/recipe &&
         conda install tvm -c ./conda/pkg
+    - name: Build iOS RPC@MacOS
+      if: startsWith(matrix.os, 'macOS')
+      run: |
+        IOS_VERSION = "14.0"
+        CMAKE_FLAGS="-DCMAKE_BUILD_TYPE=Release \
+                     -DCMAKE_SYSTEM_NAME=iOS \
+                     -DCMAKE_SYSTEM_VERSION=${IOS_VERSION} \
+                     -DCMAKE_OSX_SYSROOT=iphonesimulator \
+                     -DCMAKE_OSX_ARCHITECTURES=x86_64 \
+                     -DCMAKE_OSX_DEPLOYMENT_TARGET=14.0 \
+                     -DCMAKE_BUILD_WITH_INSTALL_NAME_DIR=ON \
+                     -DUSE_IOS_RPC=ON"
+
+        mkdir build-ios-simulator
+        cd build-ios-simulator
+        cmake .. ${CMAKE_FLAGS}
+        cmake --build . --target ios_rpc
```

Comment:
Should we check the status of each command in this script? Like "&&" at the end of each line..

Reply:
No, you don't need to do this in this script. If something goes wrong, the whole process will stop on the line of the script where the failure occurred.